docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,191 @@
+Regarding pod offloading, remote namespaces host the offloaded pods belonging to the home twin namespace, as if they were executed in the home cluster.
+
+### Quick Offloading
+
+Liqo provides a label-based mechanism to quickly set-up namespace offloading over remote clusters.
+To enable it, you should label a target namespace:
+
+```bash
+kubectl label namespace target-namespace liqo.io/enabled=true
+```
+
+Pods scheduled in the "*target-namespace*" will be potentially offloaded inside remote clusters. 
+With *quick offloading*, you are selecting all peered clusters suitable for offloading. 
+If you need a fine-grained approach, you can rely on custom offloading and the *NamespaceOffloading* resource.
+
+### Custom Offloading
+
+To control all different aspects of the namespace extension, Liqo provides a NamespaceOffloading resource. 
+The policies defined inside the NamespaceOffloading object specify how the local namespace can be replicated on peered clusters.
+
+Any namespace inside the home cluster can have its own NamespaceOffloading resource and its corresponding offloading policy.
+In other words, the NamespaceOffloading object defines "*per-namespace boundaries*", limiting the scope where pods can be remotely offloaded.
+
+As presented in the following example, the NamespaceOffloading resource is composed of three main fields:
+
+{{% render-code file="static/examples/namespace-offloading-default.yaml" language="yaml" %}}
+
+{{% notice warning %}}
+The name of the resource must be always "*offloading*" to ensure the uniqueness of a single configuration for
+each local namespace. A resource created with a different name will not trigger the topology creation.
+{{% /notice %}}
+
+#### Selecting the namespace mapping strategy
+
+The namespaceMappingStrategy defines the naming strategy used to create the remote namespaces. 
+The accepted values are:
+
+| Value               | Description |
+| --------------      | ----------- |
+| **DefaultName** (Default)    | The remote namespaces have the name of the local namespace followed by the local cluster-id to guarantee the absence of conflicts. |
+| **EnforceSameName** | The remote namespaces have the same name as the namespace in the local cluster (this approach can lead to conflicts if a namespace with the same name already exists inside the selected remote clusters). |
+
+{{% notice info %}}
+The DefaultName value is recommended if you do not have particular constraints related to the remote namespaces name. 
+However, using the DefaultName policy, the namespace name cannot be longer than 63 characters according to the [RFC 1123](https://datatracker.ietf.org/doc/html/rfc1123). 
+Since the cluster-id is 37 characters long, the home namespace name can have at most 26 characters.
+{{% /notice %}}
+
+#### Selecting the pod offloading strategy
+
+The podOffloadingStrategy defines constraints about pod scheduling.
```

Comment:
Should podOffloadingStrategy be pascal case? Same for the other resource fields.

Reply:
Yes, we could use pascal case, thank you.